src/test/java/net/datafaker/service/FakeValuesServiceTest.java

```diff
@@ -27,7 +27,8 @@
 import java.time.format.DateTimeFormatter;
 import java.time.format.DateTimeFormatterBuilder;
 import java.time.temporal.ChronoField;
-import java.util.*;
+import java.util.List;
+import java.util.Locale;
```

Comment:
thank you :wink:

Reply:
You are welcome :) 

P.S. Such import changes are usually done automatically by IDE.

Reply:
Ya I know, but it's a nit pick of mine so I'm happy to see it :wink: